### ISSUE TITLE
When removing last element of input data, Pager is blank

### ISSUE THREAD

**Issue by NeverwinterMoon** [fermoya/SwiftUIPager#16]
```swift
@State var notifications = [
  "You have run out of steam in your engine, buddy!",
  "Your dragon is feeling healthy",
  "This is a long, long tale of magic and heroes. So sit down and listen carefully."
]

var body: some View {
  VStack(spacing: 16) {
    if notifications.count > 0 {
      Pager(data: Array(0..<notifications.count), id: \.self) { index in
        self.bannerView(self.notifications[index])
          .onTapGesture {
            self.notifications.remove(at: index)
          }
      }
        .itemSpacing(8)
        .itemAspectRatio(0.9)

    }
  }
}
```

If I click on second message, everything works fine. If I click on 1st message, everything works fine. If I click one by one from first message to last, everything works as expected. But if I scroll to the last element and click on it, the whole Pager view becomes blank.

Working as expected:
[![Image from Gyazo](https://i.gyazo.com/86d210326b530f0427f6ab22d61e56a8.gif)](https://gyazo.com/86d210326b530f0427f6ab22d61e56a8)

Not working as expected:
[![Image from Gyazo](https://i.gyazo.com/b6b74ad6b508d2907e08b07dabf8a0f9.gif)](https://gyazo.com/b6b74ad6b508d2907e08b07dabf8a0f9)

**Comment by fermoya** [comment]
Hi @NeverwinterMoon ,

It seems SwiftUI recreates `Pager` but keeps its old state, therefore page was out of bounds. Can you try using version 1.3.1-beta.1? Should be fixed there.

If you need me to release this beta on Cocoapods, please let me know.

Thanks

**Comment by NeverwinterMoon** [comment]
I tested it a bit and it works great, will try it more throughly today! CocoaPods release would certainly be great. Thanks a lot!

**Comment by fermoya** [comment]
Hi @NeverwinterMoon ,

I can release the beta in Cocoapods if you're using it to help you test it. I'll wait a bit to bump the version in case something else arises.

Thanks a lot

**Comment by NeverwinterMoon** [comment]
Well, I can use it though CocoaPods like this without a release anyway `pod 'SwiftUIPager', :git => 'https://github.com/fermoya/SwiftUIPager.git', :branch => 'fix/pager-swiping'`, so I guess the actual beta release is not that crucial.

I've first tested everything in a playground project and it looked fine, now I carried over to testing this in my actual project and having no issues whatsoever except for the scrolling one mentioned at #18. That one is a real bummer, considering that the pager frame occupies plenty of screen that is also vertically scrollable. But I will close this ticket, as the original problem seems to be completely resolved. And thanks a lot, great work!
